wpengine/atlas-content-modeler
# BH-909: Reorder fields via keyboard input

- Implements accessible field reordering via keyboard input.
- Speaks the new order of fields to screen reader users.
- Persists field order to the database if no order updates have been made for 1 second.

### To test
1. Click or give keyboard focus to the reorder button for a field.
2. Press the up and down arrow keys to reorder that field.

After 1 second, refresh the page (causes field data to be refetched from the DB). Field order should have saved and you will see fields in the same order you left them.

### Out of scope
- Drag and drop to be implemented separately.
- Styling updates for field focus to be made later as part of general design tweaks.

<img width="1097" alt="Screenshot 2021-03-19 at 09 49 19" src="https://user-images.githubusercontent.com/647669/111774631-04bc4100-88b0-11eb-8729-e2ce0a665816.png">
